apps/SampleSite/config.js

```diff
@@ -103,6 +110,7 @@ case "cdn.rawgit.com":
             "amcharts.plugins.responsive": "//" + window.location.hostname + "/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/plugins/responsive/responsive",
             "amcharts.plugins.dataloader": "//" + window.location.hostname + "/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/plugins/dataloader",
             "amchartsImg": "//" + window.location.hostname + "/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/images/",
+            "simpleheat": "https://raw.githubusercontent.com/mourner/simpleheat/gh-pages/simpleheat.js",
```

Comment:
Should be:  "//" + window.location.hostname + "/mourner/simpleheat/gh-pages/simpleheat.js" which allows http + https support.